include/geo_commands.h

```diff
@@ -469,4 +470,24 @@ enum GeoLayoutCommands {
 #define GEO_CULLING_RADIUS(cullingRadius) \
     CMD_BBH(GEO_CMD_NODE_CULLING_RADIUS, 0x00, cullingRadius)
 
+/**
+ * 0x21: Create cylindrical billboarding node, rotating to match the camera's
+ * look direction.
+ *   0x01: u8 params
+ *      0b1000_0000: if set, enable displayList field and drawingLayer
+ *      0b0100_0000: if set, use the direction from the camera to the node instead
+ *      0b0000_1111: drawingLayer
+ *   0x02: s16 xAxis
+ *   0x04: s16 yAxis
+ *   0x06: s16 zAxis
+ *   0x08: [u32 displayList: if MSbit of params is set, display list segmented address]
+ */
+#define GEO_CYLINDRICAL_BILLBOARD_WITH_DL(layer, useNodePos, ax, ay, az, displayList) \
+    CMD_BBH(GEO_CMD_NODE_CYLINDRICAL_BILLBOARD, (0x80 | (useNodePos << 6) | layer), ax), \
```

Comment:
Even though this only takes up 1 bit, I would prefer to not compromise on max number of layers here (even if it means making the command larger). While this hasn't actually been addressed yet, eventually the goal is for HackerSM64 to safely support up to 128 layers.